content/api/api-alerts.md

```diff
@@ -0,0 +1,180 @@
+---
+title: "Alerts API"
+draft: false
+categories:
+tags: ["#api", "#alerts"]
+author: Lawrence Lane
+alwaysopen: false
+pre: ""
+---
+
+[Alerts][1] group anomalous behavior and are informed by multi-conditional policies you select or build in CloudWisdom. Through the Alerts API, you can:
+
+- Get a list of alerts, filterable by properties
+- Close an alert via ID
+- Get an alert via ID
+
+## Parameters
+
+| Parameter | Parameter Type | Data Type | Description |
+|-------------|----------------|-----------|--------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| policyId | Query | String | Each alert is tied to a policy. The policyID can be found in the URL, such as https://app.metricly.com/#/alerts/fda75bec-b25a-4e7f-b714-64ceb0976f2c?tab=all |
+| elementId | Query | String | An alert's scope can be one or many elements; use the elementId to filter results. |
+| elementName | Query | String | An alert's scope can be one or many elements; use the elementName to filter results. |
+| startDate | Query | Date-Time | The first moment the policy conditions were met; YYYY-MM-DDTHH:MM:SSZ |
+| endDate | Query | Date-Time | The last moment the policy conditions were met; YYYY-MM-DDTHH:MM:SSZ |
+| isClosed | Query | Boolean | When the alert was closed by a user; YYYY-MM-DDTHH:MM:SSZ |
+| limit | Query | Integer | Query object return limit. Increase to return more results. |
+| offset | Query | Integer | Number of objects skipped in the query. |
+| incidentId | Path | String | Used to DELETE or close alerts; found in response body of alert queries. |
+
+## GET an Alerts List from /incidents
```

Comment:
Could we follow the header formats in the other docs like these here? 
https://docs.metricly.com/api/api-metrics/#get
https://docs.metricly.com/api/api-metrics/#get-from-metrics

![image](https://user-images.githubusercontent.com/7016497/70811958-ac38f280-1d94-11ea-8521-988b7caf0f24.png)

Reply:
The feedback I got was that organizing them in that way did not match the order of swagger and was confusing. If i order them by GET/PUT/DELETE etc, then these for example will be re-arranged 
![image](https://user-images.githubusercontent.com/25370251/70813192-1a7eb480-1d97-11ea-935f-e5bb6744e0cb.png)

Reply:
We agreed to order by the order in Swagger during a meeting.